### PR TITLE
Babel commonjs bugfix #2568

### PR DESCRIPTION
**Pull request by eusorov**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Bugfix #2568 

* **What is the current behavior?** (You can also link to an open issue here)
see #2568 

* **What is the new behavior (if this is a feature change)?**
the plugin transforms commonjs to ES2015. In older browsers no js confilcts anymore.

* **Other information**:
it is sunny outside

### CODE REVIEW

**Comment by eusorov** [comment]
The first commit (server for webfaction) was not intended to be here ... how can I delete it from PR?

File: web/server.js [hunk 1]
```diff
@@ -116,7 +116,7 @@ app
 
 server.timeout = config.api.timeout || 120000;
 server.on('request', app.callback());
-server.listen(config.api.port, config.api.host, '::', () => {
+server.listen({port : config.api.port} , () => {
```
Comment (askmike) [inline web/server.js]: why is this change necessary?

**Review by askmike — COMMENTED** [review]
question ^

**Comment by askmike** [comment]
Thanks a ton for this PR, while I was working on 0.6 this problem made me pull my hair out haha. I was unable to find this plugin.

File: web/server.js (hunk 1 above)
Comment (eusorov) [reply web/server.js]: Somehow there were problems to get it work, so I tried different things. And somehow it got to the main develop branch. 

You can ignore it completely.

**Comment by eusorov** [comment]
I deleted this PR and make a new one without other changes

**Comment by askmike** [comment]
Okay, thanks!
